src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  return function(firstNumber) {
+    return function(secondNumber) {
+      callCount++;
+
+      return callCount <= 3 || callCount % 2 !== 0
+        ? firstNumber + secondNumber
+        : 'Bzzz... Error!';
```

Comment:
[CODE STYLE] - Don't nest ternary operators. It's better to use if-else statements for clarity and maintainability of the code.